### PR TITLE
Lazy load redis asyncio.Lock

### PR DESCRIPTION
**Pull request by jsoucheiron**
Lazy load asyncio.Lock on redis backend. This fixes an issue with multiprocess apps like tornado that might fork after import time.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.9.1 (2018-04-25)
+
+* Lazy load asyncio.Lock on redis backend. This fixes an issue with multiprocess
+apps like tornado that might fork after import time. - Jordi Soucheiron
```
Comment (argaen) [inline CHANGELOG.md]: No need to add this, the changelog is generated automatically whenever there is a new release

File: aiocache/_version.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '0.9.0'
+__version__ = '0.9.1'
```
Comment (argaen) [inline aiocache/_version.py]: Same

**Comment by jsoucheiron** [comment]
Reverted

**Comment by argaen** [comment]
Thanks! will do a 0.9.1 release this afternoon :)

**Pull request merged**